pkg/proxy/iptables/proxier.go

```diff
@@ -989,7 +1003,16 @@ func (proxier *Proxier) syncProxyRules() {
 				} else {
 					socket, err := proxier.portMapper.OpenLocalPort(&lp)
 					if err != nil {
-						glog.Errorf("can't open %s, skipping this externalIP: %v", lp.String(), err)
+						msg := fmt.Sprintf("can't open %s, skipping this externalIP: %v", lp.String(), err)
+
+						proxier.recorder.Eventf(
+							&api.ObjectReference{
+								Kind:      "Port",
```

Comment:
Port is not a kind. This is not discoverable from kubectl. The object reference should be to the service object.